core-processor/src/executor.rs

```diff
@@ -335,7 +334,7 @@ pub fn execute_wasm<A: ProcessorExt + EnvExt + IntoExtInfo + 'static, E: Environ
     };
 
     // Page which is right after stack last page
-    log::trace!("Stack end page = {:?}", stack_end_page);
+    log::trace!("Stack end page = {stack_end_page:?}");
```

Comment:
Saw it somewhere, but didn't find the doc which declares that such syntax works for debug print `:?`.
Does it work?

Reply:
Yes, it is even checked in compile time.